liblocation/src/test/java/com/mapbox/android/core/location/LocationEngineResultTest.java

```diff
@@ -36,6 +42,85 @@ public void passValidIntent() {
     assertThat(result.getLastLocation()).isSameAs(location);
   }
 
+  @Test
+  public void passNullLocation() {
+    Location location = null;
```

Comment:
Sounds good, I'd add a new line to explicitly separate `A`rrange, `A`ct and `A`ssert but that's personal preference, I think it reads better. 

👀 [_Working Effectively with Unit Tests_](https://leanpub.com/wewut/read) 
> when writing tests you should prefer DAMP (Descriptive And Maintainable Procedures)